_config.yml

```diff
@@ -317,218 +317,204 @@ navigation:
+              internal: true
+          - text: 18F labs
+            children:
+            - text: Writing Lab
+              url: writing-lab/
+              internal: true
+            - text: Design Lab
+              url: https://github.com/18F/design-lab
+              internal: false
+            - text: Dev Lab
+              url: https://gsa-tts.slack.com/messages/CDE5DU7QC/
+              internal: false
+          - text: 18F locations
+            children:
+            - text: Distributed
+              url: distributed/
+              internal: true
+            - text: Chicago
+              url: chicago/
+              internal: true
+            - text: New York City
+              url: new-york-city/
+              internal: true
+            - text: San Francisco
+              url: san-francisco/
+              internal: true
+            - text: Washington, D.C.
+              url: washington-dc/
+              internal: true
+          - text: 18F platforms
+            children:
+            - text: Cloud.gov
+              url: https://cloud.gov/
+              internal: false
+            - text: Login.gov
+              url: https://login.gov/
+              internal: false
+          - text: 18F team resources
+            children:
+            - text: Before you ship
+              url: https://before-you-ship.18f.gov/infrastructure/aws/
+              internal: false
+            - text: Blogging
+              url: blogging/
+              internal: true
+            - text: Doing research at 18F
+              url: research-guidelines/
+              internal: true
+            - text: Working on an acquisition engagement
+              url: working-on-an-acquisition-engagement/
+              internal: true
+              children:
+              - text: Acquisition engagement types
+                url: acquisition-engagement-types/
+                internal: true
+              - text: Acquisition engagement glossary
+                url: working-on-an-acquisition-engagement/#acquisition-glossary
+                internal: true
+        - text: Centers of Excellence (CoE)
+          children:
+          - text: Overview of CoE
+            url: centers-of-excellence/
+            internal: true
+          - text: CoE Website
+            url: https://coe.gsa.gov/
+            internal: false
+        - text: Outreach
+          children:
+          - text: Outreach
+            url: outreach/
+            internal: true
+          - text: Social media
+            url: social-media/
+            internal: true
+        - text: Presidential Innovation Fellows (PIF)
+          children:
+          - text: PIF Website
+            url: https://presidentialinnovationfellows.gov/
+            internal: false
+  - text: Office of Solutions
     children:
-    - text: PIF Website
-      url: https://presidentialinnovationfellows.gov/
-      internal: false
+        - text: Office of Products and Programs (OPP)
```

Comment:
Wait, OPP isn't a child of Solutions, is it? Shouldn't references to OPP become "Solutions"?